Doc/library/platform.rst

```diff
@@ -272,6 +272,10 @@ Unix Platforms
 
    .. versionadded:: 2.6
 
+   .. versionchanged:: 3.8
```

Comment:
I don't think that it's correct to use ".. versionchanged:: 3.8" tag in Python 2.7. Just add a regular paragraph before ".. versionadded:: 2.6".

Reply:
Done